mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -1195,6 +1195,14 @@ def Arith_ExtFOp : Arith_FToFCastOp<"extf"> {
   }];
   let hasVerifier = 1;
   let hasFolder = 1;
+  let hasCanonicalizer = 1;
+
+  let arguments = (ins FloatLike:$in, DefaultValuedAttr<
+                       Arith_FastMathAttr,
+                       "::mlir::arith::FastMathFlags::contract">:$fastmath);
```

Comment:
I don't think contract should be the default here. This is the exact sort of breaking change we're trying to avoid by making this opt-in and not opt-out.